bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml

```diff
@@ -299,6 +315,18 @@
 		<item-type>String</item-type>
 		<label>Vacuum Room [room#]</label>
 	</channel-type>
+	<channel-type id="collectdust" advanced="true">
```

Comment:
Shouldn't this be:
```suggestion
	<channel-type id="collect_dust" advanced="true">
```

for staying consistent with the binding naming convention?